include/mc_control/fsm/TransitionMap.h

```diff
@@ -28,6 +28,11 @@ namespace fsm
  * - ["InitState", "OK", "GraspBar", "Strict"]
  * - ["GraspBar", "OK", "LiftLeftFoot", "StepByStep"]
  * - ["GraspBar", "NOK", "GraspBar", "Auto"]
+ * - ["GraspBar", "DEFAULT", "GraspBar", "Auto"]
+ *
+ * If the "DEFAULT" output is present, this transition will be returned in case
+ * no other output pattern has been matched. If "DEFAULT" is ommited, the
+ * transition has no meaning.
```

Comment:
This is needlessly confusing indeed, I think it's enough to say:

> If the "DEFAULT" output is present, this transition will be returned in case no other output pattern has been matched. 

If it's ommitted, it simply means that there will be no transition in case none of the pattern match:

```
[A, Outpu1, B],
[A, Output2, C]
```
State A outputs "Output3", then there is no valid transition.

```
[A, Output1, B],
[A, Output2, C]
[A, DEFAULT, DefaultState]
```
If state A outputs "Output3" (or anything other than "Output1" or "Output2"), it transitions to `DefaultState`